packages/graphql/lib/getDefaultResolvers/MutationBuilder/resolveUpdateMutation.ts

```diff
@@ -2,25 +2,31 @@ import { IMutationViewMeta } from "@fullstack-one/schema-builder";
 import { IParsedResolveInfo } from "../types";
 import { IMutationBuildObject, IMutationInputObject } from "./types";
 import parseValue from "./parseValue";
+import { ReturnIdHandler } from "../../ReturnIdHandler";
 
-export default function resolveUpdateMutation(query: IParsedResolveInfo<IMutationInputObject>, mutation: IMutationViewMeta): IMutationBuildObject {
+export default function resolveUpdateMutation(
+  query: IParsedResolveInfo<IMutationInputObject>,
+  mutation: IMutationViewMeta,
+  returnIdHandler: ReturnIdHandler
+): IMutationBuildObject {
   const values: string[] = [];
 
   const fieldAssignments: string = Object.keys(query.args.input)
     .filter((fieldName) => fieldName !== "id")
     .map((fieldName, index) => {
       const value: any = query.args.input[fieldName];
-      values.push(parseValue(value));
+      values.push(parseValue(value, returnIdHandler));
```

Comment:
> `values.push(parseValue(value, returnIdHandler));`

Did you mean:
`values.push(parseValue(value, returnIdHandler.getReturnId(???)));`